tests/integration/security/rbac_v1beta1_test.go

```diff
@@ -285,3 +285,51 @@ func TestV1beta1_WorkloadSelector(t *testing.T) {
 			rbacUtil.RunRBACTest(t, cases)
 		})
 }
+
+// TestV1beta1_RequestHeaders tests v1beta1 authorization with "request.headers".
+func TestV1beta1_RequestHeaders(t *testing.T) {
```

Comment:
Why do we have a special case for this? This seems like just a normal case, should this be added as a case in an existing test rather than a whole new test?

Reply:
I didn't find a good existing test case to add this one, I will try do refactor this a little bit with more tests in master.